charts/datahub/subcharts/datahub-gms/templates/deployment.yaml

```diff
@@ -84,14 +84,14 @@ spec:
           {{- end}}
           livenessProbe:
             httpGet:
-              path: /health
+              path: /health/check/ready
               port: http
             initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
             periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
             failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
           readinessProbe:
             httpGet:
-              path: /health
+              path: /health/check/ready
```

Comment:
This chart is pointing to v0.10.3 if I recall, does this endpoint exist there? If not, then this cannot be merged until v0.10.4